cmd/juju/resource/charmresources.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package resource
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"gopkg.in/juju/charm.v6-unstable"
+	charmresource "gopkg.in/juju/charm.v6-unstable/resource"
+	csparams "gopkg.in/juju/charmrepo.v2-unstable/csclient/params"
+
+	"github.com/juju/juju/charmstore"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// CharmResourcesCommand implements the "juju charm-resources" command.
+type CharmResourcesCommand struct {
+	baseCharmResourcesCommand
+}
+
+// NewCharmResourcesCommand returns a new command that lists resources defined
+// by a charm.
+func NewCharmResourcesCommand(resourceLister ResourceLister) modelcmd.ModelCommand {
+	var c CharmResourcesCommand
+	c.setResourceLister(resourceLister)
+	return modelcmd.Wrap(&c)
+}
+
+// Info implements cmd.Command.
+func (c *CharmResourcesCommand) Info() *cmd.Info {
+	i := c.baseInfo()
+	i.Name = "charm-resources"
```

Comment:
To follow the usual pattern, should there also be a "list-charm-resources" alias? (like for example `juju models` and `juju list-models`)